packages/facility-server/app/routes/apiv1/patient/patient.js

```diff
@@ -28,6 +28,24 @@ import { patientContact } from './patientContact';
 
 const patientRoute = express.Router();
 
+patientRoute.post(
+  '/checkDuplicates',
+  asyncHandler(async (req, res) => {
+    req.checkPermission('read', 'Patient');
+    const { models, body: patient } = req;
+
+    const potentialDuplicates = await models.Patient.sequelize.query(
+      `SELECT * FROM find_potential_patient_duplicates(:patient)`,
+      {
+        replacements: { patient: JSON.stringify(patient) },
+        type: QueryTypes.SELECT,
+      },
+    );
```

Comment:
```suggestion
    const potentialDuplicates = await models.Patient.sequelize.query(
      `SELECT * FROM find_potential_patient_duplicates(:patient)`,
      {
        replacements: { patient: JSON.stringify(patient) },
        type: QueryTypes.SELECT,
        model: models.Patient,
        mapToModel: true,
      },
    );
```
If you do this I don't think you would have to rename Object keys